test/parallel/test-http-many-ended-pipelines.js

```diff
@@ -16,7 +16,24 @@ var numRequests = 20;
 var done = 0;
 
 var server = http.createServer(function(req, res) {
-  res.end('ok');
+
+  var callbackCalled = false;
+
+  res.end('ok', function() {
```

Comment:
You can simply use `res.end('ok', common.mustCall(function() {}))`. It will fail if the function is not called exactly once.